cfme/web_ui/menu.py

```diff
@@ -493,7 +493,8 @@ def f(_):
                 el = "{} | {}".format(active_loc, inactive_loc)
                 cls._try_nav(el, toplevel, secondlevel)
 
-            nav_fn = lambda: cls._nav_to_fn(toplevel, secondlevel, reset_action, _final=True)
+            def nav_fn():
```

Comment:
this one and the next could potentially be partials as well, i think its a case where we should use a partial